isis/scripts/isis3VarInit.py

```diff
@@ -79,19 +85,35 @@ def mkdir( p ):
 mkdir( deactivate_dir )
 
 # Write the files that manage the ISIS3 environments:
-activate_vars   =   activate_dir+'/env_vars.sh'
-deactivate_vars = deactivate_dir+'/env_vars.sh'
+activate_vars_sh   =   activate_dir+'/env_vars.sh'
+deactivate_vars_sh = deactivate_dir+'/env_vars.sh'
+activate_vars_csh   =   activate_dir+'/env_vars.csh'
+deactivate_vars_csh = deactivate_dir+'/env_vars.csh'
 
-with open( activate_vars, mode='w' ) as a:
+with open( activate_vars_sh, mode='w' ) as a:
     a.write('#!/bin/sh\n')
     a.write('export ISISROOT='+      os.environ['CONDA_PREFIX']+'\n')
     a.write('export ISIS3DATA='+     args.data_dir +'\n')
     a.write('export ISIS3TESTDATA='+ args.test_dir +'\n')
-print( 'Wrote '+activate_vars )
+print( 'Wrote '+activate_vars_sh )
 
-with open( deactivate_vars, mode='w' ) as d:
+with open( deactivate_vars_sh, mode='w' ) as d:
     d.write('#!/bin/sh\n')
     d.write('unset ISISROOT\n')
     d.write('unset ISIS3DATA\n')
     d.write('unset ISIS3TESTDATA\n')
-print( 'Wrote '+deactivate_vars )
+print( 'Wrote '+deactivate_vars_sh )
+
+with open( activate_vars_csh, mode='w' ) as a:
+    a.write('#!/bin/csh\n')
+    a.write('setenv ISISROOT '+      os.environ['CONDA_PREFIX']+'\n')
+    a.write('setenv ISIS3DATA '+     args.data_dir +'\n')
+    a.write('setenv ISIS3TESTDATA '+ args.test_dir +'\n')
+print( 'Wrote '+activate_vars_csh )
+
+with open( deactivate_vars_csh, mode='w' ) as d:
+    d.write('#!/bin/csh\n')
+    d.write('unsetenv ISISROOT\n')
+    d.write('unsetenv ISIS3DATA\n')
+    d.write('unsetenv ISIS3TESTDATA\n')
+print( 'Wrote '+deactivate_vars_csh )
```

Comment:
why not just:

```python
d.write("""#!/bin/csh
unsetenv ISISROOT
unsetenv ISIS3DATA
unsetenv ISIS3TESTDATA
""")
```

Reply:
and all the other ones too. These could even just be strings at the top of the file in variables.

Reply:
Too pythonic, this is a c++ library!

Reply:
understood....merge away 😬